### ISSUE TITLE
Domain activation is Failed - [BRIEF DESCRIPTION]

### ISSUE THREAD

**Issue by uuykay** [aws-amplify/amplify-console#693]
**App Id**: dtbrp5xjorsgi
**Region**: ap-southeast-2
**Step**: Domain activation
**Status**: Failed

>**Note**: Do not include information that is sensitive in nature such as your domain name, company etc.

**Issue/question**
I am trying to add a domain with AWS Amplify but I am getting this error message:
Error message: One or more of the CNAMEs you provided are already associated with a different resource.

**Error message**
Error message: One or more of the CNAMEs you provided are already associated with a different resource.

**Note**: Be sure to check the message for sensitive information. 

**Additional information**
In the past, I have used another AWS account with AWS Amplify to use this domain. At that time, the deployment worked. This time, with my current account, it won't let me. I have since deleted all resources and even deleted the old account. The domain I am trying to add is educats.com.au

**Comment by Athena96** [comment]
Hi @uuykay
These guides are helpful for common custom domain issue. If those guides can not solve your issues, please let us know.
https://docs.aws.amazon.com/amplify/latest/userguide/custom-domains.html#cnamealreadyexistsexception-error

https://aws.amazon.com/premiumsupport/knowledge-center/resolve-cnamealreadyexists-error/

**Comment by uuykay** [comment]
Hello @Athena96 I used these two guides when I have first tried to solve this problem. I don't have premium support, so I cannot message Amazon, but I have created a TXT record that matches my CNAME. Are you able to help me?

**Comment by joshqq** [comment]
Hey... check if you have the app created in another region. I have created the app in region A and another in Region B without me knowing. It passes all SSL etc until the last steps and keep telling me cname has already been used.

**Comment by uuykay** [comment]
Hey @joshqq I only have 1 Amplify App on this account. In the past I used the same domain on a different AWS account using Amplify, but I have deleted that account and all resouces (as far as I know) from that account 2 days ago. I want to be using the current account. Could you please check on it? https://www.educats.com.au/

**Comment by joshqq** [comment]
This is what I see...
<img width="975" alt="image" src="https://user-images.githubusercontent.com/27541954/82724360-0f499080-9d08-11ea-81fe-93168307e957.png">

I have to go to all the "region" and see if I have used amplify to create an app.
I can't tell you exactly how to resolve it completely but the following:
1. Ensuring the DNS in Domain Registered and Zone are the same.
2. Make sure that I have no CloudFront instances that I did not realise I have.
3. Remove SES / DKIM set up just in case.

I empathise with what you go through... let's hope you find out the root cause...

**Comment by uuykay** [comment]
Hey @joshqq thanks so much for your help. I had to go to work and couldn't reply earlier. You were actually right with your first comment about a possible instance of the app being created previously in a different region. I am not totally sure how this happened - I may have set that one up in the past and forgot about it - but it was in the Ohio region.

I discovered this when I went to my S3 buckets and found one bucket in the Ohio region with the same name as my app. Upon changing regions with Amplify I discovered and deleted that instance of the app and was able to finally link my Sydney region to the domain.

I'm very thankful!
